pages/api/ipfs/index.ts

```diff
@@ -0,0 +1,137 @@
+import { IOMarketMetadata } from "@zeitgeistpm/sdk";
+import type { PageConfig } from "next";
+import { NextRequest } from "next/server";
+
+const CLUSTER_ENDPOINT = process.env.IPFS_CLUSTER_URL;
+
+const headers = {
+  Authorization: `Basic ${Buffer.from(
+    `${process.env.IPFS_CLUSTER_USERNAME}:${process.env.IPFS_CLUSTER_PASSWORD}`,
+  ).toString("base64")}`,
+};
+
+export const config: PageConfig = {
+  runtime: "edge",
+};
+
+export default async function handler(req: NextRequest) {
+  if (req.method === "POST") {
+    const [error, parsed] = IOMarketMetadata.validate(
```

Comment:
It ensures that the form of the json matches what we expect as valid market metadata?

Reply:
Yes